pkg/api/user.go

```diff
@@ -33,7 +33,7 @@ func getUserUserProfile(userId int64) Response {
 
 // GET /api/users/loginoremail/:loginOrEmail
 func GetUserByLoginOrEmail(c *middleware.Context) Response {
-	query := m.GetUserByLoginQuery{LoginOrEmail: c.Params(":loginOrEmail")}
+	query := m.GetUserByLoginQuery{LoginOrEmail: c.QueryEscape("loginOrEmail")}
```

Comment:
not sure QueryEscape is the right method to use here, it HTML encodes the email, the email is not stored html encoded